packages/postgres-database/src/simcore_postgres_database/models/user_to_projects.py

```diff
@@ -4,18 +4,33 @@
 from .projects import projects
 from .users import users
 
+
+# DEPRECATED!!!!!!!!!!!!!! DO NOT USE!!!!!!!
 user_to_projects = sa.Table(
     "user_to_projects",
     metadata,
     sa.Column("id", sa.BigInteger, nullable=False, primary_key=True),
     sa.Column(
         "user_id",
         sa.BigInteger,
-        sa.ForeignKey(users.c.id),  # TODO: , ondelete="CASCADE"),
+        sa.ForeignKey(
+            users.c.id,
+            name="fk_user_to_projects_id_users",
```

Comment:
This class is only needed for migration, right?

Reply:
right so, I read from an article in medium [that one](https://benchling.engineering/move-fast-and-migrate-things-how-we-automated-migrations-in-postgres-d60aba0fc3d4) from @Surfict  about migrating databases.

removing a table or column has to follow this:
- first remove all occurences of its usage in the code
- deploying the new versions shall work, and also the old versions will still work since we did not remove the table/column -> this is useful in case of rollback or parallel deployment, ideally mark as deprecated. ideally the column shall continue to be updated if needed to ensure rolling back is possible.
- after we are sure we will never roll back anymore, delete the table/column

Reply:
This sounds like a smart way of doing it!